Formula/s/sf.rb

```diff
@@ -0,0 +1,25 @@
+class Sf < Formula
+  desc "Command-line toolkit for Salesforce development"
+  homepage "https://developer.salesforce.com/tools/salesforcecli"
+  url "https://registry.npmjs.org/@salesforce/cli/-/cli-2.61.8.tgz"
+  sha256 "6d7388565f83c15edd2ad05d1f42737de64680b8ea18a7f569435e93e00d6620"
+
+  license "BSD-3-Clause"
```

Comment:
you dont need space.

```suggestion
  license "BSD-3-Clause"
```

Reply:
you dont need empty line for it.